src/core/lwip/tcp.c

```diff
@@ -94,7 +94,6 @@ u16_t lwip_tcp_mss = CONST_TCP_MSS;
 u8_t enable_push_flag = 1;
 u8_t enable_ts_option = 0;
 u32_t lwip_tcp_snd_buf = 0;
-u32_t lwip_zc_tx_size = 0;
```

Comment:
The field is removed but i can't find the usage to be removed.

Reply:
Usage is removed in the following commit:
```
issue: 3788369 Pass iovec to tcp_write_express()
```
AI: Need to reorder commits and update the commit message that the option is unused.

Reply:
Done. Usage removed in the right commit now.